### PR TITLE
[!!!][BUGFIX] Check if processes were successful

### PR DESCRIPTION
**Pull request by andreaskienast**
Symfony's Process class swallows errors from executed commands unless
the exit state explicitly checked. This is a rather unexpected but
documented behavior, and may lead to broken releases.

All `Process` usages are now guarded by a `isSuccessful` check.

---

❗ `.env` command for `md5` has been changed - individual overrides have to updated locally

```patch
-CHECKSUM_MD5_COMMAND="md5sum %2\$s"
+CHECKSUM_MD5_COMMAND="md5sum %1\$s"
```

### CODE REVIEW

**Review by ohader — APPROVED** [review]
Looks good 👍 
But will give it a local test run first...

File: src/Command/PackageCommand.php [hunk 1]
```diff
@@ -131,16 +132,22 @@ public function execute(InputInterface $input, OutputInterface $output)
...
+        );
         unlink($artefactsDirectory . '/README_unsigned.md');
 
         // Legacy code
         // Show MD5 hashes and SHA1 hashes
-        (new Process('md5 -r *.gz *.zip', $artefactsDirectory))->run();
-        (new Process('shasum  *.gz *.zip', $artefactsDirectory))->run();
+        $this->io->note(
+            "MD5 hashes of the artefacts:\n"
+            . $this->runProcess('md5sum *.gz *.zip', $artefactsDirectory)->getOutput()
```
Comment (andreaskienast) [inline src/Command/PackageCommand.php]: Those binaries don't exist on macOS, unless installed via brew. We probably need some kind of a "fallback chain" here...

Comment (ohader) [reply src/Command/PackageCommand.php]: Spotted that as well... actually we'd have `CHECKSUM_MD5_COMMAND` in `.env` for that...

Comment (ohader) [reply src/Command/PackageCommand.php]: Ah, it was hard-coded to `md5` before. That's why it worked...

File: src/Command/PackageCommand.php [hunk 2]
```diff
@@ -131,16 +132,22 @@ public function execute(InputInterface $input, OutputInterface $output)
...
+        );
+        $this->io->note(
+            "SHA1 hashes of the artefacts:\n"
+            . $this->runProcess('sha1sum *.gz *.zip', $artefactsDirectory)->getOutput()
```
Comment (ohader) [inline src/Command/PackageCommand.php]: Overlooked this one... `shasum` since we're using different algos

**Review by ohader — CHANGES_REQUESTED** [review]
Hashsum generation binaries via `.env`

**Review by ohader — APPROVED** [review]

**Comment by ohader** [comment]
cc @andreasfernandez @bmack Please make sure to update your local `.env` concerning `md5` command

**Pull request merged**